src/main/resources/templates/java/exercise/git.ignore.file

```diff
@@ -81,4 +81,9 @@ nbdist/
 .idea
 *.iws
 *.iml
-out
\ No newline at end of file
+out
+
+
+#################
+.gradle
+gradle
```

Comment:
Yes, you are right. It actually does not make a difference because this file is overwritten in case Gradle is chosen as the project type for the template/solution repositories. But for the sake of consistency, I removed it.